### PR TITLE
Chat/rooms

### PR DESCRIPTION
**Pull request by Leo-Gyger**

### CODE REVIEW

File: backend/scripts/initdb.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const { Client } = pkg;
 
 const createdb = async () => {
   const client = new Client({
-    user: "postgres",
```
Comment (Nogesma) [inline backend/scripts/initdb.js:6]: User should stay postgres

File: frontend/example.env [hunk 2]
```diff
@@ -1,3 +0,0 @@
-VITE_CLIENT_ID=<id>
```
Comment (Nogesma) [inline frontend/example.env:1]: This file should not be deleted

File: frontend/src/lib/Navbar.svelte [hunk 3]
```diff
@@ -1,31 +0,0 @@
-<script lang="ts">
```
Comment (Nogesma) [inline frontend/src/lib/Navbar.svelte:1]: This file should not be deleted

File: frontend/src/lib/ProfilePic.svelte [hunk 4]
```diff
@@ -1,21 +0,0 @@
-<script lang="ts">
```
Comment (Nogesma) [inline frontend/src/lib/ProfilePic.svelte:1]: This file should not be deleted

File: frontend/src/routes/chat/Chat.svelte [hunk 5]
```diff
@@ -1,79 +0,0 @@
-<script lang="ts">
```
Comment (Nogesma) [inline frontend/src/routes/chat/Chat.svelte:1]: It's ok to delete this but we will need to refer to it in the future for the css

**Review by Nogesma — CHANGES_REQUESTED** [review]

File: frontend/src/routes/chat/Chat.svelte (hunk 5 above)
Comment (Leo-Gyger) [reply frontend/src/routes/chat/Chat.svelte:1]: fixed

**Review by Nogesma — DISMISSED** [review]

File: frontend/src/lib/ProfilePic.svelte (hunk 4 above)
Comment (Leo-Gyger) [reply frontend/src/lib/ProfilePic.svelte:1]: fixed

**Review by Nogesma — APPROVED** [review]

**Pull request merged**